supertokens_python/recipe/emailpassword/utils.py

```diff
@@ -302,14 +309,29 @@ def validate_and_normalise_user_input(recipe: EmailPasswordRecipe, app_info: App
                                       reset_password_using_token_feature: Union[
                                           InputResetPasswordUsingTokenFeature, None] = None,
                                       email_verification_feature: Union[InputEmailVerificationConfig, None] = None,
-                                      override: Union[InputOverrideConfig, None] = None) -> EmailPasswordConfig:
+                                      override: Union[InputOverrideConfig, None] = None,
+                                      email_delivery_config: Union[EmailDeliveryIngredient, None] = None
+                                      ) -> EmailPasswordConfig:
     if override is None:
         override = InputOverrideConfig()
     if reset_password_using_token_feature is None:
         reset_password_using_token_feature = InputResetPasswordUsingTokenFeature()
 
     if sign_up_feature is None:
         sign_up_feature = InputSignUpFeature()
+
+    email_service = email_delivery.service if email_delivery_config is not None else None
+    if email_service is None:
+        # assert config.create_and_send_custom_email is not None, "Shouldn't have happened"
+        email_service = BackwardCompatibilityService(
+            app_info,
+            # reset_password_using_token_feature,
+            # email_verification_feature,
+            # email_delivery_config
+        )
+
+        email_delivery = TypeInput[TypeEmailPasswordEmailDeliveryInput](email_service)
+
```

Comment:
All of this will go inside a function and you return that function

Reply:
Yeah. Fixed